src/registrar/fixtures.py

```diff
@@ -116,6 +116,12 @@ class UserFixture:
             "username": "5dc6c9a6-61d9-42b4-ba54-4beff28bac3c",
             "first_name": "David-Analyst",
             "last_name": "Kennedy-Analyst",
+        }
```

Comment:
```suggestion
        },
```
One more missing comma